helpers/osinfo.go

```diff
@@ -220,10 +221,12 @@ func (btfi *OSInfo) discoverOSDistro() error {
 	return nil
 }
 
-func FtraceEnabled() bool {
+func FtraceEnabled() (bool, error) {
 	b, err := os.ReadFile("/proc/sys/kernel/ftrace_enabled")
-	if err != nil || len(b) != 2 || b[0] == '0' {
-		return false
+	b = bytes.TrimSpace(b)
```

Comment:
this should be after the error check, no ?

Reply:
My mistake. Also added a check for length to avoid an awkward go panic